### DISCUSSION TITLE
[Projeto Angular] Documentação de componentes

### DISCUSSION THREAD

**Discussion by magnolemos** [frontendbr/forum#1456] [Discussão]
Galera, bão demais?

Projeto que estou está ficando grande e, como muito componentes estamos fazendo na mão, temos que documentar esses caras.

Dessa forma, queria saber a experiencia de vocês. O que estão usando para documentar, e quanto tempo de desenvolvimento vocês gastam para documentar? 
Ex: uma tarefa de 10 horas, quantas horas vocês gastam documentando?

**Comment by fabiomirandaa** [comment]
Infelizmente estou em projeto com prazo surreal. Mas, você pode tentar usar algo como isso https://compodoc.app/

**Comment by magnolemos** [comment]
Bacana demais @fabiomirandaa. esse cara parece massa!

Meu problema maior por hora é saber quanto tempo a mais de dev a equipe irá gastar para ter a maioria dos componentes documentados. Assim fico mais confiante para bater nessa tecla aqui. To sentindo falta, mas gostaria da experiencia de alguém para aplicar essa cultura aqui.
